packages/frontend/app/components/app/sidebar/actions/response-panel/index.hbs

```diff
@@ -0,0 +1,11 @@
+<button class='button button-xs' type='button' {{on 'click' this.yes}}>
+	<span class='icon'>{{fa-icon 'check'}}</span>
```

Comment:
Indentation should be two spaces here instead of tab?
(I think the .editorconfig specifics this?)